docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:16.04
+RUN apt-get clean && apt-get update && apt-get install -y locales tzdata
+RUN locale-gen en_US.UTF-8
+RUN update-locale
+RUN mkdir /usr/src/app
+# Set your timezone
+RUN ln -fs /usr/share/zoneinfo/Europe/Moscow /etc/localtime && dpkg-reconfigure -f noninteractive tzdata
```

Comment:
This doesn't look very generic to me... Is it?

Reply:
I agree. Del